worldedit-core/build.gradle.kts

```diff
@@ -49,7 +53,9 @@ dependencies {
     implementation("com.github.luben:zstd-jni:1.4.9-5")
     compileOnly("net.fabiozumbi12:redprotect:1.9.6")
     api("com.github.intellectualsites.plotsquared:PlotSquared-API:4.514") { isTransitive = false }
-    api("com.plotsquared:PlotSquared-Core:5.13.11") { isTransitive = false }
+    api("com.plotsquared:PlotSquared-Core:6.0.0-SNAPSHOT") { isTransitive = false }
+    api("net.kyori:adventure-api:4.8.0")
+    api("net.kyori:adventure-text-minimessage:4.1.0-SNAPSHOT")
```

Comment:
`compileOnlyApi` should be fine here. We don't need to export adventure transitively at runtime; P2 already does that.